### PR TITLE
Homepage redesign

### PR DESCRIPTION
**Pull request by parndt**
This is a work in progress.  

Instead of hardcoded companies, we have 5 random companies and 3 random individuals displayed on the homepage.

I feel that this helps to drive home the fact that Ruby Together is Ruby *Together* by being more inclusive about the list of names that we display as supporters to first time visitors.

I've also restyled the buttons as call to action buttons so that they're more visible.  For some insight into this, you can look at [this copyhackers guide](https://copyhackers.com/the-9-point-checklist-button/)

I've also wrapped the homepage quote in an HTML blockquote tag.

Feedback welcomed!

### CODE REVIEW

**Comment by indirect** [comment]
Screenshots would be great. Better yet, can you set up Heroku pipeline for this so we can just see what the app will look like in every PR?

**Comment by parndt** [comment]
> Better yet, can you set up Heroku pipeline for this so we can just see what the app will look like in every PR?

on it!

**Comment by parndt** [comment]
I have restructured the homepage to include the funding thermometer right at the start, and other changes. It'd be best to [look at the review app](https://rubytogether-staging-pr-29.herokuapp.com/).

I changed "Join Ruby Together" to "Back Ruby Together". Will this cause any problems?

**Comment by indirect** [comment]
Looking good! Let's update the roadmap now that we're featuring it. (I'm assuming this is a partial and these changes will apply to the homepage and the roadmap page).

File: app/views/home/_funding_levels.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<section>
+  <h3>Fundraising Progress</h3>
+
+  <%= render "funding_progress_bar" %>
+
```
Comment (indirect) [inline app/views/home/_funding_levels.html.erb]: I think maybe insert something like this... not sure if before or after the infographic is better.

```
In our first year, Ruby Together has completed significant improvements to the infrastructure that Ruby developers use every day:

- Shipped Bundler versions 1.10, 1.11, and 1.12, featuring conditional gems, optional groups, and a new gem index that can scale to handle the next five years of gems.
- Launched Gemstash 1.0, a server that can cache gems for your datacenter or office or host your private gems.
- Migrated RubyGems.org to use CDN servers around the world to serve data, speeding up gems for everyone.
```

File: app/views/home/_funding_progress_bar.html.erb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  <section class="progress-bar-entry">
+    <p class="amount">$35-50k</p>
+    <p class="title">Steady Improvements</p>
+    <ul>
+      <li>Hire a full-time dev to improve gem tools</li>
+      <li>Finish the new index project in just a few weeks (3-6 weeks)</li>
+      <li>Migrate all of RubyGems.org to use the Fastly CDN (4-8 weeks)</li>
+      <li>Build a local gem-caching server for datacenters and offices (4 months)</li>
+      <li>Create a public Ruby ecosystem analytics dashboard (1 month)</li>
```
Comment (indirect) [inline app/views/home/_funding_progress_bar.html.erb]: Let's replace these bullet points with something like:

- Hire a full-time dev to improve gem tools
- Create a public Ruby ecosystem analytics dashboard
- Merge Bundler and RubyGems into a single, unified project
- Expand RubyBench to comprehensively cover Ruby, Rails, and Bundler

Maybe there's more, but I'm not sure.

File: app/views/home/_funding_levels.html.erb (hunk 1 above)
Comment (parndt) [inline app/views/home/_funding_levels.html.erb]: I've added this, and put it before the infographic.  What do you think of its placement now that you can [see it on the page](https://rubytogether-staging-pr-29.herokuapp.com)?

**Comment by indirect** [comment]
That's great. Once the bullets inside the roadmap are updated, I am 👍 on this.

File: app/views/home/_funding_progress_bar.html.erb (hunk 2 above)
Comment (parndt) [inline app/views/home/_funding_progress_bar.html.erb]: Good 👍

**Comment by parndt** [comment]
@indirect I have made you 👍  on this

**Comment by indirect** [comment]
hooray :tada: let's do it.

**Pull request merged**